cypress/integration/e2e/roadmap-page.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Roadmap Page', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('/page-data/roadmap/page-data.json').as('roadmapDataRequest');
```

Comment:
I'm using the interceptor here because AFAIK:
1. if all went well at build time, this route should respond with the JSON data used to generate the static page
2. So we can check that it's not empty (e.g. there was a problem with the query and it did not return the expected content), as well as that we are rendering the expected count of release tables, and release names.